README.md

```diff
@@ -1,118 +1,80 @@
-# Nextcloud Swarm Plugin
+# **Hejbit - Nextcloud Swarm Plugin**
 
-_Decentralized Sovereign cloud-storage comes to Nextcloud!_
+*Bring decentralized, sovereign cloud storage to Nextcloud with the Hejbit Swarm plugin!*
 
-This plugin brings [Swarm](https://www.ethswarm.org/) to [Nextcloud](https://nextcloud.com) - decentralized file-storage, connected to the blockchain.
+This plugin integrates [Swarm](https://www.ethswarm.org/) (a decentralized, blockchain-connected file storage system) directly into your [Nextcloud ](https://nextcloud.com/)instance.
 
-![View file and action menu](assets/images/swarm_Files.png "View file and action menu")
-
-<em>File view when using the plugin in Nextcloud</em>
+![File view when using the plugin in Nextcloud](https://github.com/user-attachments/assets/24f89fd9-f8eb-47e3-97ac-e1da41e48072)
 
 ## Table of Contents
 
--   [Dependencies](#dependencies)
--   [Setup and Installation](#setup-and-installation)
--   [Usage](#usage)
-    -   [Settings](#settings)
-    -   [Using Swarm](#using-swarm)
--   [License](#license)
+- [Dependencies](#dependencies) .
+- [Setup and Installation](#setup-and-installation) .
+- [Usage](#usage) . 
+  - [Settings](#settings) .
```

Comment:
@frozenbanana The point after the link was only a fix to prevent opening the thumbnail on collectives. You can remove it from here.